installer/conf/omi_mapping.json

```diff
@@ -219,7 +219,8 @@
             },
             {
                 "CimPropertyName": "UsedMemory",
-                "CounterName": "Used Memory"
+                "CounterName": "Used Memory",
```

Comment:
In order to maintain backward and forward user compatibility; since users may have searches that query on the "Used Memory" CounterName, I needed a way to keep the same returned data for them, but still show on the portal "Used Memory kBytes".